### PR TITLE
Tagging policy

### PR DESCRIPTION
**Pull request by bwlang**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
@bwlang you need to bump the tool version. Thanks

**Comment by bwlang** [comment]
I can't find a dup in the ignore tag of picard_ValidateSamFile.xml

anybody see what I'm missing?
```
Applying linter inputs... FAIL
.. ERROR: Select parameter [ignore] has multiple options with the same text content
```

File: tools/picard/picard_ValidateSamFile.xml [hunk 1]
```diff
@@ -80,61 +80,61 @@
...
+      <option value="MISMATCH_MATE_ALIGNMENT_START">MISMATCH_MATE_ALIGNMENT_START</option>
+      <option value="MATE_FIELD_MISMATCH">MATE_FIELD_MISMATCH</option>
+      <option value="INVALID_TAG_NM">INVALID_TAG_NM</option>
+      <option value="MISSING_TAG_NM">MISSING_TAG_NM</option>
+      <option value="MISSING_HEADER">MISSING_HEADER</option>
+      <option value="MISSING_SEQUENCE_DICTIONARY">MISSING_SEQUENCE_DICTIONARY</option>
+      <option value="MISSING_READ_GROUP">MISSING_READ_GROUP</option>
+      <option value="RECORD_OUT_OF_ORDER">RECORD_OUT_OF_ORDER</option>
+      <option value="READ_GROUP_NOT_FOUND">READ_GROUP_NOT_FOUND</option>
+      <option value="RECORD_MISSING_READ_GROUP">MISSING_READ_GROUP</option>
```
Comment (nsoranzo) [inline tools/picard/picard_ValidateSamFile.xml:110]:
```suggestion
      <option value="RECORD_MISSING_READ_GROUP">RECORD_MISSING_READ_GROUP</option>
```

**Review by nsoranzo — COMMENTED** [review]
Found it!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @bwlang

**Comment by bwlang** [comment]
whew - a bit more involved than the oneliner I anticipated - thanks for the help @nsoranzo and @bgruening

**Pull request merged**